core/src/main/scala/com/salesforce/op/evaluators/OpForecastEvaluator.scala

```diff
@@ -48,8 +51,10 @@ import com.twitter.algebird.macros.caseclass
  * @param isLargerBetter is metric better if larger
  * @param uid            uid for instance
  */
+
 private[op] class OpForecastEvaluator
 (
+  val seasonalWindow: Int = 1,
```

Comment:
1. please update the docs explaining what the `seasonalWindow` param represents, it's accepted range of values etc.
2. add `require` checks for `seasonalWindow` range in the initialization block of `OpForecastEvaluator` class